packages/providers/src.ts/formatter.ts

```diff
@@ -397,21 +397,22 @@ export class Formatter {
 
     responseFromRecord(record: HederaTransactionRecord): TransactionResponse {
         return {
-            chainId: record.chainId,
-            hash: record.hash,
-            timestamp: record.timestamp,
-            transactionId: record.transactionId,
-            from: record.from,
-            to: record.to,
-            data: record.call_result,
-            gasLimit: BigNumber.from(record.gas_limit),
-            value: BigNumber.from(record.amount),
+            chainId: record.chainId ? record.chainId : null,
+            hash: record.hash ? record.hash : null,
+            timestamp: record.timestamp ? record.timestamp : null,
+            transactionId: record.transactionId ? record.transactionId : null,
+            from: record.from ? record.from : null,
```

Comment:
Are the checks for `hash`, `timestamp` and `from` necessary if we are populating them in every case. We should remove the optional flag to them

Reply:
Done